packages/site-middleware/src/plugins/prettyCode.ts

```diff
@@ -1,4 +1,11 @@
+import { getHighlighter } from 'shiki';
+
 const options = {
+  getHighlighter: highlighterOptions =>
+    getHighlighter({
+      ...highlighterOptions,
+      langs: ['bat', 'diff', 'css', 'json', 'jsx', 'tsx', 'mdx', 'yaml', 'ts']
```

Comment:
I don't think we use any other languages in code samples on the site...

@lilyvc

Reply:
maybe js? for a config.js

Reply:
Good shout